src/sensitivities/chainrules.jl

```diff
@@ -0,0 +1,197 @@
+#using InteractiveUtils
+
+function generate_overload(sig)
```

Comment:
I would add in a comment here explaining what cases we're ignoring and returning `false` for. It's a bit overwhelming reading all of these cases and trying to understand it.

Reply:
urg, rebasing has eaten a bunch of comments,
Yes, should do that, also should make it actually return false, rather than nothing for cases it isn't generating anything for